### PR TITLE
Better cli error visibility

### PR DESCRIPTION
**Pull request by RoiGlinik**
add common errors page to docs. 
improve cli error visibility.
add help to the ssl cli issue

### CODE REVIEW

File: src/robusta/cli/main.py [hunk 1]
```diff
@@ -42,6 +41,13 @@
 RUNNER_CONFIG_FOR_SMALL_CLUSTERS = "512Mi"
 GRAFANA_RENDERER_CONFIG_FOR_SMALL_CLUSTERS = "64Mi"
 
+# catch unhandled exceptions.
+def my_except_hook(exctype, value, traceback):
+    typer.echo("Please use —debug flag for more info")
```
[the remark this answers is not in the document]
Comment (aantn) [reply src/robusta/cli/main.py:46]: This is just for errors that we're not catching anywhere else, right?

File: docs/common-errors.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    :color: light
+
+    1. Determine where the Robusta-cli binary file is installed  
+
+    .. code-block:: bash
+        :name: find-python-bin-loc
+
+        which python3
+    
+    this <new-path> should be added to the user path directores.
```
Comment (aantn) [inline docs/common-errors.rst:18]: Capital `T` in `this`

File: docs/common-errors.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        source <shell-config-file>
+
+.. dropdown:: CLI SSL certificate issue
+    :color: light
+
+    This error may suggest a python certificate package is missing in your system.
+    Try running the script located at:
+    /Applications/Python 3.9/Install Certificates.command
+
+    for more info see:
```
Comment (aantn) [inline docs/common-errors.rst:41]: Capital `F` in `for`

File: src/robusta/cli/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (RoiGlinik) [reply src/robusta/cli/main.py:46]: yep , just for unhandled exceptions

**Pull request merged**